apis/python/src/tiledb/vector_search/ingestion.py

```diff
@@ -2532,21 +2532,19 @@ def consolidate_and_vacuum(
             tiledb.vacuum(parts_uri, config=conf)
             tiledb.consolidate(ids_uri, config=conf)
             tiledb.vacuum(ids_uri, config=conf)
-        group.close()
-
-        # TODO remove temp data for tiledb URIs
-        if not index_group_uri.startswith("tiledb://"):
-            group = tiledb.Group(index_group_uri, "r")
-            if PARTIAL_WRITE_ARRAY_DIR in group:
-                group.close()
-                group = tiledb.Group(index_group_uri, "w")
-                group.remove(PARTIAL_WRITE_ARRAY_DIR)
-                vfs = tiledb.VFS(config)
-                partial_write_array_dir_uri = (
-                    index_group_uri + "/" + PARTIAL_WRITE_ARRAY_DIR
-                )
-                if vfs.is_dir(partial_write_array_dir_uri):
-                    vfs.remove_dir(partial_write_array_dir_uri)
+        if PARTIAL_WRITE_ARRAY_DIR in group:
+            group.close()
```

Comment:
Let's move this above the `if` block and use `with` consistently whenever possible?

Reply:
Applied